tests/Integration/SmushSubscriberTestCase.php

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
This TestCase could be in the same folder as the tests for Smush as it's a very specific use case

Reply:
Agree, but it will trigger a "Cannot redeclare class" error then.

Reply:
Surprisingly, no error ¯\(°_o)/¯